package.json

```diff
@@ -47,7 +47,7 @@
     "classnames": "^2.1.3",
     "lodash": "^3.10.1",
     "react": "^0.13.0",
-    "react-redux": "^0.2.2",
-    "redux": "^1.0.0-rc"
+    "react-redux": "^3.0.1",
```

Comment:
I updated `react-redux` because it supports binding action creators now so you don't need to do that by hand.